lib/types.js

```diff
@@ -133,3 +150,31 @@ TDouble.prototype = {
 };
 
 exports.TDouble = TDouble;
+
+function TTrue (value) { this.value = value; }
+TTrue.prototype = {
+    typetag: 'T',
+    decode: function (data) {
+        this.value = true;
+        return data;
+    },
+    encode: function (buf, pos) {
+        return pack('', buf, pos, []);
```

Comment:
have you tested this? The jspack docs specify certain supported formats, of which true and false are not one of them. Does calling the method this way work?